README.md

```diff
@@ -914,6 +914,8 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
   * CouchDB [[couchdb](https://crates.io/keywords/couchdb)]
     * [chill-rs/chill](https://github.com/chill-rs/chill) [[couchdb](https://crates.io/crates/chill)] — A Rust client for the CouchDB REST API [<img src="https://api.travis-ci.org/chill-rs/chill.svg?branch=master">](https://travis-ci.org/chill-rs/chill)
     * [Sofa](https://github.com/66Origin/sofa) — an interface to CouchDB HTTP REST API for stable rust
+  * [DynamoDB](https://aws.amazon.com/dynamodb) [[dynamodb](https://crates.io/keywords/dynamodb)]
```

Comment:
Also this should be `https://aws.amazon.com/dynamodb/` (note the ending slash) as the other link 301's to that

Reply:
Wow, thanks :D